content/vendor/guides/cli-quickstart.md

```diff
@@ -89,44 +90,26 @@ SEQUENCE    CREATED    EDITED    ACTIVE_CHANNELS
 
 ### 3. Getting some YAML
 
-Next, we'll need some YAML files from which to create a release.
-We'll start from the repo at https://github.com/replicatedhq/replicated-starter-kots but you can use any set of Kubernetes YAMLs or a Helm Chart.
+Next, we'll need some YAML files from which to create a release, we'll start from the repo at https://github.com/replicatedhq/replicated-starter-kots.
```

Comment:
Not grammatically correct. Maybe use ; to join if you don't want them being separate sentences?

Reply:
```suggestion
Next, we'll need some YAML files from which to create a release.
We'll start from the repo at https://github.com/replicatedhq/replicated-starter-kots.
```